### ISSUE TITLE
Why was the callback for the suffix buttons removed?

### ISSUE THREAD

**Issue by emaq-farishadziomerovic** [salim-lachdhaf/dropdown_search#730]
The commit [e6cdd6c](https://github.com/salim-lachdhaf/dropdown_search/commit/e6cdd6c1b74c41924dd9f1300d61528512e143a7) removed the onPressed callback from the IconButtonProps class and I dont think it was mentioned in any breaking change.

What is the reason for it's removal?
Can it be returned?

**Comment by git-goku** [comment]
hi, any update on this

**Comment by git-goku** [comment]
the solution is onChanged: (newValue) {}, handle clear if newValue is null

**Comment by emaq-farishadziomerovic** [comment]
Yeah, but it was possible to abort the cancel or do whatever you want with the onPressed callback.

Anyways I've stopped using this package because you can't open the menu/dialog/sheet without using the widget. 

I think that would be a great addition to this package, to expose public methods for showing the menu.

**Comment by salim-lachdhaf** [comment]
this could help for use of DropDownSearch without visible drop down field: https://github.com/salim-lachdhaf/dropdown_search/issues/714
